src/State.php

```diff
@@ -30,11 +30,14 @@ enum State: string
     public function getCountry(): Country
     {
         $key = "alldressed-{$this->value}-country";
+        $value = $this->value;
```

Comment:
Pourquoi tu met dans une variable ?

Reply:
Pour pouvoir le passer en paramètre dans le `use ()` à la ligne 36. Est-ce que il y a un autre alténative pour utiliser `$this->value` dans le `App::singleton()` ?

Reply:
Ça ne fonctionne pas parce que ton callback est `static`. Tu peux simplement enlever le `static` et tu peux utiliser `$this` normalement.